src/iadmin.cpp

```diff
@@ -147,6 +149,140 @@ auto get_replica_value(
     return v;
 } // get_replica_value
 
+auto check_that_connected_user_is_rodsadmin(RcComm* _comm) -> bool
```

Comment:
`RcComm&`?

Reply:
Sure

Reply:
This function has been removed. Resolving.